quic/s2n-quic-core/src/recovery/rtt_estimator.rs

```diff
@@ -251,9 +251,11 @@ impl RttEstimator {
         //# smoothed_rtt = 7/8 * smoothed_rtt + 1/8 * adjusted_rtt
         //# rttvar_sample = abs(smoothed_rtt - adjusted_rtt)
         //# rttvar = 3/4 * rttvar + 1/4 * rttvar_sample
-        self.smoothed_rtt = weighted_average(self.smoothed_rtt, adjusted_rtt, 8);
+        //#
+        //# this logic has been updated to follow the errata reported in https://www.rfc-editor.org/errata/eid7539
```

Comment:
The `//#` is specifically for RFC compliance comments that get parsed by the Duvet tool, so this line should just use regular `//` syntax on a separate line

```suggestion
        
        // this logic has been updated to follow the errata reported in https://www.rfc-editor.org/errata/eid7539
```

Reply:
OK. I will update the code.